openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -32,6 +38,111 @@ def get_identity(self, data):
             return None
 
 
+class UserProfileReadOnlySerializer(serializers.Serializer):
```

Comment:
I agree.  Definitely a bit confusing giving that it doesn't just serialize the `UserProfile`model.  This serializes both.  Any thoughts on a better name?

`UserProfileUserReadOnlySerializer`
`CombinedAccountReadOnlySerializer`
`JointProfileReadOnlySerializer`  ?

/cc @andy-armstrong